tests/test_validation.py

```diff
@@ -93,3 +93,29 @@ def test_qlora_merge(self):
 
         with pytest.raises(ValueError, match=r".*4bit.*"):
             validate_config(cfg)
+
+    def test_hf_use_auth_token(self):
+        base_cfg = DictDefault(
```

Comment:
I think you don't need base_cfg here and the merge below. It was useful in others because I wanted to inherit similar base configs.